### PR TITLE
multiple changes to make document easier to read

### PR DESCRIPTION
**Pull request by Mohit-Nain**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/loki.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Examples:
 
 The [same rules that apply for Prometheus Label Selectors](https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors) apply for Loki Log Stream Selectors.
 
-Another way to add a label selector, is in the table section, clicking on the **Filter** button beside a label will add the label to the query expression. This even works for multiple queries and will the label selector to each query.
+Another way to add a label selector, is in the table section, clicking on the **Filter** button beside a label will add the label to the query expression. This even works for multiple queries and will add the label selector to each query.
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/loki.md:101]:
```suggestion
Another way to add a label selector is in the table section. Click **Filter** beside a label to add the label to the query expression. This even works for multiple queries and will add the label selector to each query.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/features/datasources/loki.md [hunk 2]
```diff
@@ -43,12 +43,12 @@ The Derived Fields configuration allows you to:
 * Add fields parsed from the log message. 
 * Add a link that uses the value of the field. 
 
-You can use this functionality to link to your tracing backend directly from your logs, or link to a user profile page if a userId is present in the log line. These links will be shown in the [log details](/features/explore/#labels-and-parsed-fields).
+You can use this functionality to link your tracing backend directly from your logs, or link a user profile page if a userId is present in the log line. These links will be shown in the [log details](/features/explore/#labels-and-parsed-fields).
```
[the remark this answers is not in the document]
Comment (aknuds1) [reply docs/sources/features/datasources/loki.md:46]: I think this change is wrong, the original wording should be kept.

File: docs/sources/features/datasources/loki.md [hunk 3]
```diff
@@ -43,12 +43,12 @@ The Derived Fields configuration allows you to:
...
 {{< docs-imagebox img="/img/docs/v65/loki_derived_fields.png" class="docs-image--no-shadow" caption="Screenshot of the derived fields configuration" >}}
 Each derived field consists of:
 
 - **Name:** Shown in the log details as a label.
-- **Regex:** A Regex pattern that runs on the log message and captures part of it to as the value of the new field. Can only contain capture a single group.
+- **Regex:** A Regex pattern that runs on the log message and captures part of it as the value of the new field. Can only contain pattern to capture a single group.
```
Comment (aknuds1) [inline docs/sources/features/datasources/loki.md:51]:
```suggestion
- **Regex:** A Regex pattern that runs on the log message and captures part of it as the value of the new field. Can only contain a single capture group.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: docs/sources/features/datasources/loki.md [hunk 4]
```diff
@@ -43,12 +43,12 @@ The Derived Fields configuration allows you to:
 * Add fields parsed from the log message. 
 * Add a link that uses the value of the field. 
 
-You can use this functionality to link to your tracing backend directly from your logs, or link to a user profile page if a userId is present in the log line. These links will be shown in the [log details](/features/explore/#labels-and-parsed-fields).
+You can use this functionality to link your tracing backend directly from your logs, or link a user profile page if a userId is present in the log line. These links appear in the [log details](/features/explore/#labels-and-parsed-fields).
```
Comment (aknuds1) [inline docs/sources/features/datasources/loki.md:46]:
```suggestion
You can use this functionality to link to your tracing backend directly from your logs, or link to a user profile page if a userId is present in the log line. These links appear in the [log details](/features/explore/#labels-and-parsed-fields).
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]

**Review by aknuds1 — APPROVED** [review]
LGTM

**Review by oddlittlebird — APPROVED** [review]
LGTM. Thanks for improving this doc!

**Pull request merged**